### PR TITLE
fix(Core/AuraScript): Soul Feast should apply on some Valithria mobs

### PR DESCRIPTION
**Pull request by Kitzunu**
## CHANGES PROPOSED:
-  Allow Shadow's Edge to add Soul Feast stacks to some Vali adds

```
    GLUTTONOUS_ABOMINATION = 37886,
    RISEN_ARCHMAGE         = 37868,
    BLISTERING_ZOMBIE      = 37934,
    BLAZING_SKELETON       = 36791
```  

## TESTS PERFORMED:
- Builds
- Tested in-game

## HOW TO TEST THE CHANGES:

1. .tel icecrowncitadel
2. Enter and spawn the mobs `.npc add temp ###`
3. Hit them with your Shadow's edge `.add 49888`
4. When you have the quest A feast of souls `.q add 24547`

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: src/server/scripts/Spells/spell_item.cpp [hunk 1]
```diff
@@ -2530,15 +2538,38 @@ class spell_item_unsated_craving : public SpellScriptLoader
         {
             PrepareAuraScript(spell_item_unsated_craving_AuraScript);
 
+            bool ExceptionMobs(Unit* target) const
```
Comment (FrancescoBorzi) [inline src/server/scripts/Spells/spell_item.cpp:2541]: a better name would be `bool isException(Unit* target) const`

Comment (Kitzunu) [reply src/server/scripts/Spells/spell_item.cpp:2541]: sure

**Comment by felwyn** [comment]
Fix is working! Good job to anyone included 👍 

https://i.imgur.com/89Z4zF8.jpg

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
good job @Kitzunu and @felwyn ;)

**Pull request merged**